server/server.go

```diff
@@ -230,7 +206,19 @@ func (s *Server) indexGet(w http.ResponseWriter, r *http.Request) {
 	if len(qStrs) > 0 {
 		qStr := qStrs[len(qStrs)-1]
 		t0 := time.Now()
-		results, err := s.execQuery(qStr)
+
+		var results resultSetV1
+		txn, err := s.store.NewTransaction()
+
+		if err == nil {
+			var query ast.Body
+			_, query, err = ast.CompileQuery(qStr)
+			if err == nil {
+				results, err = s.execQuery(txn, query)
+			}
+			s.store.Close(txn)
+		}
```

Comment:
If NewTransaction fails, do we return an error or an empty answer?  Seems an error would be appropriate.